pkgs/tools/misc/togglesg-download/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchFromGitHub, buildPythonApplication, makeWrapper, ffmpeg }:
+
+buildPythonApplication rec {
+
+  name = "togglesg-download-git-${version}";
+  version = "2016-02-08";
+
+  src = fetchFromGitHub {
+    owner = "0x776b7364";
+    repo = "toggle.sg-download";
+    rev = "5cac3ec039d67ad29240b2fa850a8db595264e3d";
+    sha256 = "0pqw73aa5b18d5ws4zj6gcmzap6ag526jrylqq80m0yyh9yxw5hs";
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  doCheck = false;
+  dontBuild = true;
+  dontStrip = true;
+
+  installPhase = ''
+    mkdir -p $out/bin
+    install -m755 download_toggle_video2.py $out/bin/download_toggle_video2.py
+  '';
+
+  postInstall = stdenv.lib.optionalString (ffmpeg != null)
+    ''wrapProgram $out/bin/download_toggle_video2.py --prefix PATH : "${ffmpeg}/bin"'';
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/0x776b7364/toggle.sg-download";
+    description = "Command-line tool to download videos from toggle.sg";
+    longDescription = ''
```

Comment:
This `longDescription` tells me nothing that helps me decide whether the package is of interest to me.